### ISSUE TITLE
[Bug] Sum height not working for hexbin

### ISSUE THREAD

**Issue by bnetter** [keplergl/kepler.gl#785]
**Describe the bug**
I have a CSV file with 250 transactions, including a date, a point (which is always the same), and an amount. I want to create an hexbin growing in height over time, but the height stays at 0.

**To Reproduce**
1. Download [data.csv.zip](https://github.com/keplergl/kepler.gl/files/3729801/data.csv.zip).
2. Create an hexbin, with `to_lat` as latitude, `to_lng` as longitude.
3. Activate the height, as the **sum** of the `amount`.
4. Add a filter on the `date`, and try moving the date, the height stays the same but the tooltip evolves.

**Expected behavior**
The hexbin height should grow over time.

**Screenshots**
![image](https://user-images.githubusercontent.com/336674/66840520-7cbe6300-ef68-11e9-9a1f-d47edd48ebec.png)
![image](https://user-images.githubusercontent.com/336674/66840615-a24b6c80-ef68-11e9-8386-5ef0d333dd12.png)

**Desktop (please complete the following information):**
 - OS: Mac Catalina
 - Browser: Chrome
 - Version: 1.1.9

**Comment by bnetter** [comment]
I've tried the same with a point somewhere else, as I thought it would help as a reference. It doesn't work much better. And it doesn't work much better without the sum, only with point counts.

**Comment by martimalek** [comment]
I'm also experiencing the same issue, has anyone solved this?

**Comment by martimalek** [comment]
Found a solution! It seems that sometimes kepler doesn't read the values on first load, just deleting the points/hexes and putting them again worked for me!

**Comment by bnetter** [comment]
@heshan0131 any ideas?

**Comment by martimalek** [comment]
Hey I've found the problem. Basically keplergl shows the height of a hexbin when there's a difference between the values of the current hexbins in the map. So if there are no differences it won't show height.

@bnetter in your particular example you only have one point in the map so kepler doesn't have anything to compare it with. So it always shows height 0. If you create another point with a different amount value it will show it's height correctly.

This is an example where I use one of the points in your csv and then another one I invented. I've just changed the type from point to hexbin and added the height based on **amount**.

Hope it helps.
[keplergl_fix.zip](https://github.com/keplergl/kepler.gl/files/3819315/keplergl_fix.zip)
